hack/generator/pkg/astmodel/testdata/SetIntFromInt-ViaStaging.golden

```diff
@@ -18,10 +18,16 @@ func (person Person) ConvertFrom(source *vHub.Person) error {
 	var personTemp vNext.Person
 
 	// Populate personTemp from source
-	personTemp.ConvertFrom(source)
+	err := personTemp.ConvertFrom(source)
+	if err != nil {
+		return errors.Wrap(err, "for person, calling personTemp.ConvertFrom(source)")
```

Comment:
Is this wrapping going to be too noisy if the chain of conversions has several links? In that case it seems like the message would be the same for each level. It might be better to have the package name in the message if possible, since that would be the thing that changes between each step in the chain, right? (Although that might be fiddly if it's not being passed down at the moment.)

Reply:
Yes, it will be verbose, but it seems better than not wrapping. Don't currently have the package available at this point in the code (but I think will have in a later PR), so perhaps we improve it then?